modules/tensor_mechanics/doc/content/documentation/systems/Kernels/GeneralizedPlaneStrainOffDiag.md

```diff
@@ -1,19 +1,11 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # GeneralizedPlaneStrainOffDiag
 
-!alert construction title=Undocumented Class
-The GeneralizedPlaneStrainOffDiag has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
-
 !syntax description /Kernels/GeneralizedPlaneStrainOffDiag
 
+Kernel `GeneralizedPlaneStrainOffDiag` computes the off-diagonal jacobian entries to couple the in-plane displacement degree of freedoms with the out-of-plane strain degree of freedom.
```

Comment:
Capitalize Jacobian.

Reply:
Mention here that this is typically set up by the Action, and add a link to that documentation.